src/main/java/hudson/plugins/claim/ClaimConfig.java

```diff
@@ -26,7 +31,9 @@ public ClaimConfig() {
     /**
      * Groovy script to be run when a claim is changed.
      */
+    @Deprecated
     private String groovyScript;
```

Comment:
You have a data migration logic, so it makes sense to make the field transient to prevent further serialization